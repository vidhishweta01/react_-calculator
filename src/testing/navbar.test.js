import React from 'react';
import { BrowserRouter } from 'react-router-dom';
import Navbar from '../component/navbar';
import renderer from 'react-test-renderer';

it('renders correctly', () => {
    const tree = renderer
      .create(<BrowserRouter><Navbar /></BrowserRouter>)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });