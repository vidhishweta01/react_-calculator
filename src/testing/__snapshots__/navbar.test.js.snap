// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<nav
  className="nav"
>
  <h1
    className="head"
  >
    Math Magicians
  </h1>
  <div
    className="links"
  >
    <a
      exact={true}
      href="/"
      onClick={[Function]}
    >
      Home
       
    </a>
    <a
      className="mid_link"
      href="/quote"
      onClick={[Function]}
    >
      Quote
       
    </a>
    <a
      href="/calculator"
      onClick={[Function]}
    >
      Calculator 
    </a>
  </div>
</nav>
`;
